charts/consul/Chart.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: v2
 name: consul
 version: 1.3.5
-appVersion: 1.17.5
+appVersion: 1.17.3
```

Comment:
This was a little confusing for me because there is a `v1.17.4` tag of hashicorp/consul; however, it seems this is because we didn't have our tag naming conventions in place at the time.

`v1.17.3` is the latest patch release available on homebrew and dockerhub, so I believe this is correct.